src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb

```diff
@@ -0,0 +1,119 @@
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Method", "WriteLine"),
+                                    New TaggedText("Punctuation", "("),
+                                    New TaggedText("Keyword", "string"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Parameter", "value"),
+                                    New TaggedText("Punctuation", ")"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Punctuation", "("),
+                                    New TaggedText("Punctuation", "+"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Text", "18"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Text", "overloads"),
+                                    New TaggedText("Punctuation", ")")})),
+                            QuickInfoSection.Create("DocumentationComments",
+                                ImmutableArray.Create({New TaggedText("Text", "Writes the specified string value, followed by the current line terminator, to the standard output stream.")})),
+                            QuickInfoSection.Create("Exception",
+                                ImmutableArray.Create({
+                                    New TaggedText("Text", "Exceptions"),
+                                    New TaggedText("LineBreak", "\r\n"),
+                                    New TaggedText("Space", " "),
+                                    New TaggedText("Namespace", "System"),
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Namespace", "IO"),
+                                    New TaggedText("Punctuation", "."),
+                                    New TaggedText("Class", "IOException")}))}))
+
+            Dim view = New Mock(Of ITextView) With {
+                .DefaultValue = DefaultValue.Mock
+            }
+
+            Dim textVersion = view.Object.TextSnapshot.Version
+            Dim trackingSpan = textVersion.CreateTrackingSpan(codeAnalysisQuickInfoItem.Span.ToSpan(), SpanTrackingMode.EdgeInclusive)
+
+            Dim intellisenseQuickInfo = IntellisenseQuickInfoBuilder.BuildItem(trackingSpan, codeAnalysisQuickInfoItem)
+
+            Assert.NotNull(intellisenseQuickInfo)
+
+            Assert.IsType(Of Adornments.ContainerElement)(intellisenseQuickInfo.Item)
+            Dim container = CType(intellisenseQuickInfo.Item, Adornments.ContainerElement)
+            Assert.Equal(3, container.Elements.Count())
+            Assert.Equal(ContainerElementStyle.Stacked, container.Style)
+
+
+            Assert.IsType(Of Adornments.ContainerElement)(container.Elements.ElementAt(0))
```

Comment:
Assert.IsType returns the thing already casted. So you can write:

Dim firstRowContainer = Assert.isType(....)(container.Elements.ElementAt(0))

And combine this and the next line into one.

Reply:
done